common/src/main/scala/com/gu/akka/implicits/package.scala

```diff
@@ -0,0 +1,11 @@
+package com.gu.akka
```

Comment:
I'm not massively keen on bringing Akka into the project, is it just handling http? Could we use OkHttp and the [WebServiceHelper](https://github.com/guardian/support-workers/blob/master/common/src/main/scala/com/gu/helpers/WebServiceHelper.scala) class in the same way as [ZuoraService](https://github.com/guardian/support-workers/blob/master/common/src/main/scala/com/gu/zuora/ZuoraService.scala) or [SalesforceService](https://github.com/guardian/support-workers/blob/master/common/src/main/scala/com/gu/salesforce/SalesforceService.scala) do?